packages/blockchain-wallet-v4-frontend/src/components/Flyout/model.tsx

```diff
@@ -182,7 +194,7 @@ const getActionText = (action: ActionEnum, date: string | number) => {
           id='scenes.coin.recurringbuy.next_swap_on_date'
           defaultMessage='Next Swap on {date}'
           values={{
-            date: moment(date).format('ddd, MMMM Do')
+            date: date.format('ddd, MMMM Do')
```

Comment:
to avoid duplication of formatting strings, do the formatting once at the top of the component after line 139.  then just pass as `values={{ date }}`

Reply:
yup makes sense. new commit incoming.